algobot/enums.py

```diff
@@ -13,8 +14,56 @@
 LONG = 1
 SHORT = -1
 
-TRAILING = 2
-STOP = 1
+
+class LossStrategy:
+    TRAILING = 2
+    STOP = 1
+
+    @staticmethod
+    def from_str(value: str) -> int:
+        if value.lower() == "trailing":
+            return LossStrategy.TRAILING
+        elif value.lower() == "stop":
+            return LossStrategy.STOP
+        else:
+            ValueError(f"{value} is unsupported")
+
+    @staticmethod
+    def to_str(loss_strategy: Optional[int]) -> str:
+        if loss_strategy == LossStrategy.STOP:
+            return "Stop"
+        elif loss_strategy == LossStrategy.TRAILING:
+            return "Trailing"
+        elif loss_strategy is None:
+            return "None"
+        else:
+            raise ValueError(f"Unknown type {loss_strategy}")
+
+
+class ProfitType:
```

Comment:
This seems nicer, although there is so much duplication :/

Reply:
Let's refactor to:

```
Class OrderType:
    TRAILING = 2
    STOP = 1
    ...
```

Come to think of it, we don't really need both of these :)

Reply:
Makes much more sense :)